storage/rocksdb/rdb_datadic.h

```diff
@@ -42,6 +42,48 @@ class Rdb_field_packing;
 class Rdb_cf_manager;
 class Rdb_ddl_manager;
 
+class Rdb_merge_operator : public rocksdb::AssociativeMergeOperator {
+ public:
+  virtual bool Merge(const rocksdb::Slice &key,
+                     const rocksdb::Slice *existing_value,
+                     const rocksdb::Slice &value, std::string *new_value,
+                     rocksdb::Logger *logger) const override {
+    
+    DBUG_ASSERT(new_value != nullptr);
+    std::string merged_value;
+
+    /* if no existing vaue, then persist the passed in value */
+    if (existing_value == nullptr) {
+      merged_value = Serialize(value);
+    } else {
+      merged_value = Deserialize(std::max(Serialize(value), Serialize(*existing_value)));
+    }
+
+    *new_value = merged_value;
+    return true;
+  }
+
+  virtual const char *Name() const override { return "Rdb_merge_operator"; }
```

Comment:
Can you add "cf_system" or something along those lines so that it's clear it's part of the system column family? Or maybe make the Rdb_merge_operator class name contain system or sys as part of its name, so it's clear this is used for the system column family?